### PR TITLE
Added the possibility to retrieve the UI labels of ingredients from the DB

### PR DESCRIPTION
**Pull request by CrimsonFreak**
As I didn't want to maintain every description of every recipe along with the Items and reward themselves, I added the possibility to retrieve the recipe desc from the DB labels with Config.UseCustomDesc.
For it to work with weapons as well, I had to make a slight modification to vorp_inventory, you'll see in the comments, surely you'll understand what I'm talking about.
Visibly my formatter isn't the same as yours, so it made changes to almost every file.
The main changes are in VUI.lua, server.lua, client.lua, app.js and Config.lua

### CODE REVIEW

**Comment by outsider31000** [comment]
all my observations were made, and also for last since if this gets changed  i still need to see what was changed in the inventory.
thanks for the pull request

**Comment by CrimsonFreak** [comment]
vorp_inventory/client/services/UtilityService.lua, Line 109:
`
Utils.GetWeaponLabel = function(hash)
	for _, wp in pairs(Config.Weapons) do
		if wp.HashName == hash then
			return wp.Name
		end
	end
	return hash
end
`

Was copy pasted to vorp_inventory/vorpInventoryApi.lua at line 232 (after self.getWeaponComponents())

`
    self.GetWeaponLabel = function(hash)
        for _, wp in pairs(Config.Weapons) do
            if wp.HashName == hash then
                return wp.Name
            end
        end
        return hash
    end
`

However, I put a check that this method exists before proceeding in the PR in server.lua if you look closely :

vorp_crafting/server/server.lua at Line 195
`
        elseif name:match("WEAPON") and VorpInv.GetWeaponLabel then
            crafts.labels[name] = { label = VorpInv.GetWeaponLabel(name), limit = 1 }
        else
`

**Comment by outsider31000** [comment]
thats fine , you need to answer to my comments i made in your pull request.

File: client/services/jobchecker.lua [hunk 1]
```diff
@@ -39,3 +39,10 @@ AddEventHandler("vorp:SelectedCharacter", function()
     appready = true
     TriggerServerEvent('vorp:findjob')
 end)
+
```
Comment (CrimsonFreak) [inline client/services/jobchecker.lua:42]: ?

**Comment by CrimsonFreak** [comment]
I'm sorry but I'm not sure what you're talking about. If you made any review or comment to what I posted, either am I blind or they're just not there on my end. For me there's nothing aside from this very discussion. 
What lines/files did you comment on ?

**Comment by outsider31000** [comment]
dont you have it as pending ? only you can add them as resolved  not sure why you cant see them

**Review by CrimsonFreak — COMMENTED** [review]
Test comment

**Comment by CrimsonFreak** [comment]
No, nothing really... I literally only see "ready to merge".

**Comment by outsider31000** [comment]
the comments to be reviewed were made on your pull request in what you changed , not in this conversation

**Pull request closed without merging**